ci/scripts/integration-test.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+require() {
+	if [ "$1" == "" ]
+	then
+		echo "The environment variable $2 must be present and contain $3"
+		exit 4
+	fi
+}
+
+section() {
+	echo -e "\n============================\n= $1\n============================\n"
+}
+
+SCRIPTDIR=$(dirname $0)
+SCRIPTDIR=$(realpath $SCRIPTDIR)
+
+section "Checking config"
+require "$SERVICE_ACCOUNT" "SERVICE_ACCOUNT" "the service account JSON gcsbeat will use"
+require "$BUCKET" "BUCKET" "the bucket name to hold the test files"
+
+section "Building gcsbeat"
+source $SCRIPTDIR/setup-go.sh
+make
+
+section "Pulling down gcloud"
+source $SCRIPTDIR/setup-gcloud.sh
+
+section "Setting up environment"
+export TESTID=`date +%s`
+export TESTDIR="/tmp/$TESTID"
+export KEYPATH="$TESTDIR/key.json"
+
+mkdir -p $TESTDIR
+
+echo "Test ID: $TESTID"
+echo $SERVICE_ACCOUNT > $KEYPATH
+ls -la $TESTDIR
+gcloud auth activate-service-account --key-file $KEYPATH
```

Comment:
The service account is activated inside the setup-gcloud.sh script. Is there a reason it's activated again here?

Reply:
I pulled out the gcloud code, and forgot this line, fixed.